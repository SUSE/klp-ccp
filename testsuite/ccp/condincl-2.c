#define A(x) x
#define B 1

#if B /* 1 */
void A(
#endif
#if B /* 2 */
#if B /* 3 */
		pu_f
#else
		undefined
#endif
#else
		undefined
#endif
#if !B /* 4 */
		undefined
#else
		(void)
#endif
#if B /* 5 */
	)
#endif
{}
