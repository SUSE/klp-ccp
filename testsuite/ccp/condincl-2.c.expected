/* klp-ccp: from ../../cp/testsuite/ccp/condincl-2.c */
#define A(x) x
#define B 1

#if B /* 1 */
void A(
#else
#error "klp-ccp: a preceeding branch should have been taken"
#endif
#if B /* 2 */
#if B /* 3 */
		klpp_pu_f
#else
#error "klp-ccp: non-taken branch"
#endif
#else
#error "klp-ccp: non-taken branch"
#endif
#if !B /* 4 */
#error "klp-ccp: non-taken branch"
#else
		(void)
#endif
#if B /* 5 */
	)
#else
#error "klp-ccp: a preceeding branch should have been taken"
#endif
{}
